website/blog/2023-04-05-ferretdb-1-0-ga.md

```diff
@@ -0,0 +1,127 @@
+
+Our mission remains the same: to provide an open-source document database with MongoDB compatibility.
+With this release, we're ecstatic to say that FerretDB is production-ready!
+
+FerretDB 1.0 GA includes all the core functionalities of a document database for production use.
+It boasts improved performance and stability over previous releases.
+You can quickly [get started here](https://docs.ferretdb.io/quickstart-guide/).
+
+## Going from Beta to FerretDB 1.0 GA
+
+With [FerretDB](https://www.ferretdb.io/), you have an easy-to-use, developer-friendly, open-source, and production-ready replacement for all your MongoDB workloads.
+
+Since our Beta release in January, we’ve added some notable features such as client TLS validation, support for aggregation pipelines, authentication for PostgreSQL, and Tigris authentication.
+In the meantime, we've also pushed more queries to the backend, optimizing them for faster performance.
+
+Along with improved database backend support for [PostgreSQL](https://www.postgresql.org/), [Tigris](https://www.tigrisdata.com/), and [SAP Hana](https://www.sap.com/products/technology-platform/hana.html), we're also working on adding other backend support in the future.
+
+In addition, we’re also improving compatibility and enabling more applications, such as [Mingo](https://mingo.io/), [DBeaver](https://dbeaver.io/), [NoSQLBooster](https://nosqlbooster.com/), and [Studio 3T](https://studio3t.com/), to leverage FerretDB functionalities.
+Please find a list of all compatible products here (Link to documentation & listed products).
+
+We are proud to announce that FerretDB now has:
+
+* 👨🏻‍💻 Over 30 code contributors with more than 130 merged pull requests from our community of contributors
+* ⭐️ over 5.3k Stars on GitHub
+* ⏫ More than 180 Docker hub downloads
+* 🔥 Over 60 running instances
+* ⏫ Over 10k FerretDB downloads
+
+Let’s look at some of the latest features and changes in this GA release.
+
+## What’s new?
+
+In this GA release, FerretDB now supports the `createIndexes` command.
+This will enable you to specify the fields you want to index, and also the type of index to use (e.g. ascending, descending, or hashed).
+
+For instance, suppose you have a `users` collection containing several fields, including "age", "name", and "email", and you want to create an index on the "age" field.
+You can now run the following command:
+
+```js
+db.users.createIndex({ age: 1 })
+```
+
+This will create an ascending index on the "age" field, which will speed up any queries that filter on that field.
+
+We've also added the `dropIndexes` command, which allows you to remove an index from a collection.
+Here's an example:
+
+```js
+db.users.dropIndex({ age: 1 })
+```
+
+This will remove the index from the "users" collection.
+
+We have expanded our aggregation pipeline functionality to include additional stages, such as `$unwind`, `$limit` and `$skip`, in addition to the `$sum` accumulator within the `$group` stage.
```

Comment:
Do we want examples for `$unwind`, `$limit` and `$skip`, and `$sum` accumulator of `$group`